client/entrypoints/payment-request-settings/payment-request-page.js

```diff
@@ -49,12 +49,12 @@ const Description = () => (
 	</>
 );
 
-const PaymentRequestsSettings = () => {
+const PaymentRequestsPage = () => {
 	return (
 		<SettingsLayout>
 			<SettingsSection Description={ Description }>
 				<LoadableSettingsSection numLines={ 30 }>
-					<PaymentRequestsCustomizer />
+					<PaymentRequestsSection />
 				</LoadableSettingsSection>
 			</SettingsSection>
 
```

Comment:
extra empty line(that's from my PR 😬 )

Reply:
I don't mind it tho - it kinda separates the `SettingsSection` from the `SaveSettingsSection` :D would you like me to remove it anyways?